core/src/main/java/io/crate/module/CrateCoreModule.java

```diff
@@ -46,6 +47,12 @@
     public CrateCoreModule(Settings settings, IndexEventListenerProxy indexEventListenerProxy) {
         logger = Loggers.getLogger(getClass().getPackage().getName(), settings);
         this.indexEventListenerProxy = indexEventListenerProxy;
+        if (settings.getAsBoolean(SharedSettings.ENTERPRISE_LICENSE_SETTING.getKey(), false) &&
```

Comment:
I think we could avoid duplicating the default value here if we used `SharedSettings.ENTERPRISE_LICENSE_SETTING.setting().get(settings)`